pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitCreationTest.java

```diff
@@ -89,6 +91,15 @@ public void createNewMockJuniperKitTest() {
                             .findAny().get();
             Assert.assertEquals(newKit.getBspCollaboratorParticipantId(), bspPrefix + "_" + participantId + rand);
             Assert.assertEquals(newKit.getBspCollaboratorSampleId(), bspPrefix + "_" + participantId + rand + "_" + kitType);
+
+            Assert.assertTrue(!(kitResponse instanceof KitResponseError));
+            Assert.assertTrue(kitResponse instanceof StatusKitResponse);
+            StatusKitResponse statusKitResponse = (StatusKitResponse) kitResponse;
```

Comment:
I mentioned this in a prior review (using class types to distinquish the return value), but needing to cast implies that your pattern can be improved.